cmd/cmd.go

```diff
@@ -344,6 +344,39 @@ func (w *progressWriter) Write(p []byte) (n int, err error) {
 	return len(p), nil
 }
 
+func loadOrUnloadModel(cmd *cobra.Command, opts *runOptions) error {
```

Comment:
Possible to update `api.md` and maybe `README.md` in a follow up PR? When this is released we can merge the docs update.